aloscene/points_2d.py

```diff
@@ -429,6 +429,35 @@ def _resize(self, size, **kwargs):
             abs_size = tuple(s * fs for s, fs in zip(size, points.frame_size))
             return points.abs_pos(abs_size)
 
+    def _rotate(self, angle, **kwargs):
+        """Rotate Point2d, but not their labels
+
+        Parameters
+        ----------
+        size : float
+
+        Returns
+        -------
+        points : aloscene.Point2d
+            rotated points
+        """
+        points = self.clone()
```

Comment:
At a first glance is seem to me that you do not check the point format (xy, or yx) neither if the points are "absolute" or "relative". Please, make sure the following is robust the differents encoding state.

Reply:
right

Reply:
Very late reply : looking back at this code the points format is not checked. Therefore you'll rotate based on W and H assuming the points are encoded as "xy" but this might not be the case.